.github/workflows/ci.yml

```
@@ -0,0 +1,32 @@
+name: test on CI
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ["2.7", "3.0", "3.1", "3.2", "head"]
+    steps:
+      - uses: actions/checkout@v2
```

I think we should stay up to date with the latest version of `checkout`, which is currently `v3`. Do you mind changing it?

```suggestion
      - uses: actions/checkout@v3
```